python/src/StandardSpaceCrossEntropyImportanceSampling_doc.i.in

```diff
@@ -0,0 +1,98 @@
+%feature("docstring") OT::StandardSpaceCrossEntropyImportanceSampling
+"Standard Space Cross-Entropy Importance Sampling.
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Parameters
+----------
+event : :class:`~openturns.ThresholdEvent`
+    Event we are computing the probability of.
+
+quantileLevel : float  :math:`0<quantileLevel<1`
+    Intermediate quantile level. The default number can be tweaked with the `CrossEntropyImportanceSampling-DefaultQuantileLevel` key from :class:`~openturns.ResourceMap`.
+    
+Notes
+-----
+The following explanations are given for a failure event defined as :math:`g(\mathbf{X})<T` with :math:`\mathbf{X}` 
+a random vector following a joint PDF :math:`f_\mathbf{X}`, :math:`T` a threshold and :math:`g` a limit state function,
+without loss of generality.
+
+The Importance Sampling (IS) probability estimate :math:`\widehat{P}^\text{IS}` is given by:
+
+.. math::
+
+    \widehat{P}^\text{IS}=\frac{1}{N} \sum_{i=1}^{N} {\mathbf{1}}_{g(\mathbf{x}_i)<T} \frac{f_\mathbf{X}(\mathbf{x}_i)}{h(\mathbf{x}_i)},
```

Comment:
Here and in other files, you may want to replace \mathbf with \vect : the macro is useful to try to make the doc consistent.

Reply:
This is now changed in StandardSpaceCrossEntropyImportanceSampling_doc.i.in and PhysicalSpaceCrossEntropyImportanceSampling_doc.i.in.